packages/wrangler/src/__tests__/pages/pages-deployment-tail.test.ts

```diff
@@ -849,14 +849,13 @@ function isRequest(event: TailEventMessageType): event is RequestEvent {
 
 /**
  * Similarly, we need to deserialize from a raw buffer instead
- * of just JSON.parsing a raw string. This deserializer also then
- * re-stringifies with some spacing, the same way wrangler tail does.
+ * of just JSON.parsing a raw string.
  *
  * @param message a buffer of data received from the websocket
- * @returns a string ready to be printed to the terminal or compared against
+ * @returns a JSON object ready to be compared against
  */
-function deserializeToJson(message: WebSocket.RawData): string {
-	return JSON.stringify(JSON.parse(message.toString()), null, 2);
+function deserializeToJson(message: WebSocket.RawData) {
```

Comment:
Since we're no longer re-serializing it **to** json:

```suggestion
function deserializeJson(message: WebSocket.RawData) {
```

or

```suggestion
function deserializeJsonMessage(message: WebSocket.RawData) {
```

Reply:
Same in the file below.